### PR TITLE
Add support for bad phrases, remove redundant entries

### PR DESCRIPTION
**Pull request by ZfsrGhS953**
Add support for bad phrases (entries consisting of more than one word), and remove redundant entries in badwords.txt.

This pull request obsоletes #1392 (i. e. there's no need to merge it as this one covers all of its changes)

### CODE REVIEW

**Comment by AlexHGaming** [comment]
hi

**Comment by MastaCoder** [comment]
I don't understand why we need to have different languages inside of the badwords file, if we just continue adding more and more it's going to get pretty damn long. However, I can leave this up for other people to decide if we should add different languages to it.

My vote is simply for someone to make another repo with multiple different badword.txt files in different languages, and if people really care they can add them all or just specific languages.

**Comment by ZfsrGhS953** [comment]
@MastaCoder That's like saying "What's the point of the non-English information on the internet? What's the point of standards like Unicode?" 
MultiOgar-Edited isn't meant to be restricted to English-speaking users. In this case, the changes allow Russian-speaking users to have a game (mostly) without swearing in chat. Before, the only way to get rid of swearing on non-English servers was to disable it altogether through ```serverChatAscii``` , forcing them to use their non-native language. This change allows Russian users to communicate in their native language, making it easier for those who are not fluent in English.
If you're concerned that it may make the file too big in size, we can make an installer with two options: full installation that installs everything (but without ads and miners please :smile: ) and partial installation, where the user can select language packs (gamemode packs, etc...)
But as of now, not a single user has complained about its half-a-meg size, so that would only be needed in the far future.

**Comment by MastaCoder** [comment]
@ZfsrGhS953 Still doesn't prove anything. Everything in this game from the clients supporting it to the information in the console are literally in English, why would we support Russian? The other code I'm fine with, it's the inclusion of the Russian swear words inside of the badwords.txt is what I'm not fine with since it correlates nothing to the base language of this server. Sure if we supported other languages and had everything else in some sort of lang file it'd make more sense. 

TLDR: It's an English made and worked on project, I don't see the point of supporting other languages as of now. I highly doubt any English speaker would care or even understand if someone is swearing in Russian. 

Also forgot to add on, it's not the half megabyte I care about, it's the fact that if this is added, it simply tells everyone that yes we support other languages and they'll begin requesting and adding different languages to the file which would just continue to increase the size.

**Comment by ZfsrGhS953** [comment]
@MastaCoder Usually the chat's users talk in a single language, not multiple ones, so the case of an English speaker and a Russian speaker being in chat is unlikely.
Yes, I think we need to support other languages. I already gave a solution to the size problem, through the use of an installer with pack selection.

**Comment by MastaCoder** [comment]
Yep and I still see no point on merging this until we get that done with.

**Comment by Luka967** [comment]
What are forks made for? Sharing valuable modifications of a repo with others. They don't need to be pushed to the parent, you know.
We're giving out a project made for the English-speaking from the beginning, and only them. Nobody else. And if we were to change that, we'd add *full* support for multiple languages from the beginning, not only two (and even that's partial). You yourself see that we don't even have good support for changing languages.
If you want to make an installer, it's yours. We're not changing our point of view unless somebody steps up the game good for others.

**Comment by ZfsrGhS953** [comment]
Pull request closed due to inactivity.

**Comment by ZfsrGhS953** [comment]
Re-opening so that the GameServer.js changes can be merged. Changes to badwords.txt rolled back.

**Comment by MastaCoder** [comment]
That's more like it.
